src/components/User/README.md

```diff
@@ -56,21 +58,28 @@ LANDING_BLOCK-->
-| className       | Custom CSS class for root element       |                              `string`                              |         |
-| style           | HTML style attribute                    |                       `React.CSSProperties`                        |         |
-| qa              | HTML `data-qa` attribute, used in tests |                              `string`                              |         |
+| Name             | Description                             |                                    Type                                     | Default |
+| :--------------- | :-------------------------------------- | :-------------------------------------------------------------------------: | :-----: |
+| avatar           | User avatar                             | [AvatarProps](../Avatar/README.md#properties) `string` `React.ReactElement` |         |
+| name             | User name                               |                              `React.ReactNode`                              |         |
+| description      | User description                        |                              `React.ReactNode`                              |         |
+| size             | User block size                         |               `'3xs'` `'2xs'` `'xs'` `'s'` `'m'` `'l'` `'xl'`               |   `m`   |
+| aria-label       | `aria-label` for user block             |                                  `string`                                   |         |
+| aria-labelledby  | `aria-labelledby` for user block        |                                  `string`                                   |         |
+| aria-describedby | `aria-describedby` for user block       |                                  `string`                                   |         |
+| aria-details     | `aria-details` for user block           |                                  `string`                                   |         |
+| className        | Custom CSS class for root element       |                                  `string`                                   |         |
+| style            | HTML style attribute                    |                            `React.CSSProperties`                            |         |
+| qa               | HTML `data-qa` attribute, used in tests |                                  `string`                                   |         |
 
 ## CSS API
 
-| Name                   | Description                      |
-| :--------------------- | :------------------------------- |
-| `--g-user-font-size`   | Name and description font size   |
-| `--g-user-line-height` | Name and description line height |
+| Name                               | Description                       |
+| :--------------------------------- | :-------------------------------- |
+| `--g-user-gap`                     | Gap between avatar and text block |
```

Comment:
What about `--g-user-avatar-offset` name for this?

Reply:
FIXED